tests/tests.js

```diff
@@ -37,16 +37,9 @@ define(['jquery', 'zimArchive', 'zimDirEntry', 'util', 'utf8'],
             var xhr = new XMLHttpRequest();
             xhr.open('GET', url);
             xhr.onload = function () {
-                if (this.status >= 200 && this.status < 300) {
-                    var blob = new Blob([xhr.response], {type: 'application/octet-stream'});
-                    blob.name = name;
-                    resolve(blob);
-                } else {
-                    reject({
-                        status: this.status,
-                        statusText: xhr.statusText
-                    });
-                }
+                var blob = new Blob([xhr.response], {type: 'application/octet-stream'});
+                blob.name = name;
+                resolve(blob);
             };
```

Comment:
It's great that it fixes the issue for Chrome, but it looses the ability to fail properly if the XHR returns an error.
Maybe accepting the 0 status code in addition to status between 200 and 300 could be enough?

The idea is to have a meaningful error in QUnit if XHR fails in the future